faq/manage-cluster-faq.md

```diff
@@ -377,17 +377,13 @@ WAL belongs to ordered writing, and currently, we do not apply a unique configur
 - NUMA: no specific suggestion; for memory allocation strategy, you can use `interleave = all`
 - File system: ext4
 
-### How is the write performance in the most strict data available mode (`sync-log = true`)?
+### Can Raft + multiple replicas in the TiKV architecture achieve absolute data safety?
 
-Generally, enabling `sync-log` reduces about 30% of the performance. For write performance when `sync-log` is set to `false`, see [Performance test result for TiDB using Sysbench](/benchmark/v3.0-performance-benchmarking-with-sysbench.md).
+Data is redundantly replicated between TiKV nodes using the [Raft Consensus Algorithm](https://raft.github.io/) to ensure recoverability should a node failure occur. Only when the data has been written into more than 50% of the replicas will the application return ACK (two out of three nodes).
 
-### Can Raft + multiple replicas in the TiKV architecture achieve absolute data safety? Is it necessary to apply the most strict mode (`sync-log = true`) to a standalone storage?
+Because theoretically two nodes might crash, data written to TiKV is flushed to the raft-log by default starting from v5.0. If a crash occurs, you can recover the KV data using the raft-log.
```

Comment:
```suggestion
Because theoretically two nodes might crash, data written to TiKV is flushed to the raft-log by default starting from v5.0. If a crash occurs, the KV data can recover from the raft-log automatically.
```